index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio MVOC</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>Welcome</h1>
+      <nav>
+        <a href="#">
+          <img src="/images/Icon-Menu.png" alt="hamburger-icon">
+        </a>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2 class="name">
+          Hey there. I'm <br> Marco
+        </h2>
+        <h2 class="ocupation">
+          I'm a software <br> developer
+        </h2>
+        <p>
+          I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+      </section>
+      <div class="social-container">
+        <ul>
+          <li><a href="#"><img src="/images/twitter.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/Linkedin-icon.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/medium.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/github.png" alt=""></a></li>
+          <li><a href="#"><img src="/images/angellist.png" alt=""></a></li>
+        </ul>
+        <a class="down-icon" href="#">
+          <img src="/images/down.png" alt="down-icon">
+        </a>
+      </div>
```

Comment:
- [ ] It is a core requirement that you align the content in this section with the flexbox. This gives us a lot of options to align all our elements, plus, it's responsive. Kindly fix that

Reply:
# Hi @cwaku,

## I made these changes:

- Turn h2 to h1 as you asked to make it the most important header in the web page.
- Add a class attribute to some tags.
- Change to flexbox the section.
- Move inside the flexbox all the content.